Apps/Src/BlinkLights.c

```diff
@@ -0,0 +1,93 @@
+#include "Tasks.h"
+#include "Lights.h"
+
+void Task_BlinkLight(void* p_arg){
+    //get necessary state pointers
+    car_state_t *car_state = (car_state_t*) p_arg;
+    blinker_states_t *blinkerStates = &(car_state -> BlinkerStates);
+    light_switches_t *storedLightSwitches = &(car_state -> SwitchStates.lightSwitches);
+    CPU_TS ts;
+    OS_ERR err;
+
+    while(1){
+        //wait for blinkLight to get signaled
+        OSSemPend( 
+            &BlinkLight_Sem4,
+            0,
+            OS_OPT_PEND_BLOCKING,
+            &ts,
+            &err
+        );    
+        //TODO: Main light toggle logic
```

Comment:
Is this TODO still accurate, or is the main light logic the code on lines 23-80?